### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by charliewilco** [Tacoma-JS/Tacoma-JS.github.io#5]
hold up, i got a few designs for this i'll implement next week

**Comment by NorthDecoder** [comment]
Go ahead with the design ideas, I am not much of an artist or aesthetistician.  I just wanted to get something up so the js.org admin would approve our DNS request.  To discourage name squatting he has a [requirement](https://github.com/js-org/dns.js.org/wiki/No-Content) that we have meaningful material posted **before** he will approve a custom DNS.

It would be wise if we could keep the design and content separated without a [CMS](https://en.wikipedia.org/wiki/Content_management_system),  maybe using markdown readers as you suggested.

**Comment by NorthDecoder** [comment]
Thanks for being interested.  I have created a branch rc_v0.1.0 [:link:](https://github.com/Tacoma-JS/Tacoma-JS.github.io/tree/rc_v0.1.0) for you to work in.  Make any changes to the web design that you want there, including deleting whatever is currently there :smiley:. That way your work will not affect or be affected by the other branch. I assume that I have set all the github permissions correctly, if not let me know.

Also if necessary, we can discuss improvements to the boilerplate contributions guideline in the readme.
